### DISCUSSION TITLE
Display live data

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46228] [Q&A] [answered]
Hello everyone ,
I have  a symfony project that extract data from server and display it on viewer but these data keep change it's not fix (like stock for example) i want to display this data and refresh it without refresh the page .
How can I do that ?
Thank you.

**Comment by alessandro-podo** [comment, marked as answer]
thats no symfony problem ..
you can a javascript to reload the data via ajax

**Comment by chyshyya** [comment]
Look for WebSocket. On server side must be some WebSocketServer. No other way to do the job gracefully.

**Comment by norkunas** [comment]
https://symfony.com/doc/current/mercure.html can do this job :)
